string field
string datatype
hmi_rosbridge/message[] value
